filebeat/datadog_checks/filebeat/filebeat.py

```diff
@@ -175,10 +174,6 @@ def __init__(self, instance):
                 "If given, filebeat's only_metrics must be a list of regexes, got %s" % (self._only_metrics,)
             )
 
-        self._timeout = instance.get("timeout", 2)
```

Comment:
Let's keep it at 2 seconds in the constructor and override the default value so we keep the same behaviour

Reply:
I added some logic to override the default of 10 set in the RequestWrapper and added a test. I'm not sure if there might be a better way or if the check should just use the request wrappers default but I'm happy to refactor if needed!